src/Containers/App/index.jsx

```diff
@@ -267,13 +269,17 @@ class App extends Component {
       const target = escs.targets[i];
       const esc = escs.individual.find((esc) => esc.index === target);
 
+      // this will throw a exception when preflight fails
```

Comment:
It would be great if you could catch the preflight error too and show a matching error message.

Reply:
this would need some error/exception refactoring, i would do that in a separate PR

Reply:
Ok, I need to look into this into more detail anyway. I don't fully understand why this can't haben in the flash method itself.

For now it is capsulated enough that it will only trigger for AM32 anyway, so it should not have any other side effects.